### PR TITLE
Add some context to reconciliation error messages

### PR DESCRIPTION
**Pull request by omertuc**
This helps track down the source of errors. Long term we're considering
having a linter for this

### CODE REVIEW

File: internal/extramanifest/extramanifest.go [hunk 1]
```diff
@@ -131,26 +131,27 @@ func (h *EMHandler) ExportExtraManifestToDir(ctx context.Context, extraManifestC
...
 
 	for i, policy := range policies {
 		objects, err := getConfigurationObjects(policy, objectLabels)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to get configuration objects: %w", err)
 		}
 		for _, object := range objects {
 			fileName := fmt.Sprintf("%d_%s_%s.yaml", i, object.GetName(), object.GetNamespace())
-			filePath := filepath.Join(toDir, PolicyManifestPath, fileName)
```
Comment (leo8a) [inline internal/extramanifest/extramanifest.go:150]: maybe removing `toDir` here accidentally?

Comment (omertuc) [reply internal/extramanifest/extramanifest.go:150]: So, to answer your question, no.

But I'll revise the name

**Pull request merged**